addons/test_mail/tests/test_performance.py

```diff
@@ -488,15 +488,15 @@ def test_mail_composer_w_template(self):
         test_record, test_template = self._create_test_records()
         test_template.write({'attachment_ids': [(5, 0)]})
 
-        with self.assertQueryCount(admin=27, employee=27):  # tm 16/16
+        with self.assertQueryCount(admin=29, employee=29):  # tm 16/16
```

Comment:
Up, I don't see any explanation on why you have additional queries in commit message (which should always be the case when changing query counters).

Cheers !

Reply:
Sorry it completely slipped my mind, it's fixed now. Don't hesitate to check if the explanation is convincing enough, my reviewer is off today :)